massband/conductivity/einstein_helfand.py

```
@@ -0,0 +1,180 @@
+class KinisiEinsteinHelfandIonicConductivity(zntrack.Node):
+    file: str | Path = zntrack.deps_path()
+    structures: list[str] = zntrack.params(default_factory=list)
+    start_dt: float = zntrack.params(10)  # in ps - start time for conductivity analysis
+    start: int = zntrack.params(0)  # in ps - start time for diffusion analysis
+    time_step: float = zntrack.params(0.5)  # in fs - time step of the simulation
+    sampling_rate: int = zntrack.params(1000)  # in fs - sampling
+
+    results: dict = zntrack.metrics()
+    data_path: Path = zntrack.outs_path(zntrack.nwd / "conductivity_data")
+
+    def run(self):
```

Ruff (0.12.2)

`run` is too complex (12 > 10)

(C901)